src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -124,7 +124,7 @@
     func (c *{@view.clientTypeName}) {@setClientInfoFunc(view)}(keyval ...string) {
         kv := append([]string{"gl-go", version.Go()}, keyval...)
         kv = append(kv, "gapic", version.Repo, "gax", gax.Version, "grpc", grpc.Version)
-        c.xGoogHeader = []string{gax.XGoogHeader(kv...)}
+        c.Metadata = metadata.Pairs("x-goog-api-client", gax.XGoogHeader(kv...))
```

Comment:
Actually I think this is wrong... will probably have to cast a `map[string][]string` instead.